explore/src/main/scala/explore/HelpBody.scala

```diff
@@ -0,0 +1,132 @@
+  private val baseUrl     =
+    base.rawUrl.addPath(List(base.user.value, base.project.value, "main") ++ path.path.init)
+  private val url         = rootUrl.addPath("main" :: path.path.toList)
+  private val rootEditUrl = base.editUrl.addPath(List(base.user.value, base.project.value))
+  private val newPage     = rootEditUrl
+    .addPath(List("new", "main") ++ (path.path.init :+ "/"))
+    .addParam("filename", path.path.last)
+    .addParam("value", s"# Title")
+    .addParam("message", s"Create $helpId")
+  private val editPage    = rootEditUrl
+    .addPath(List("edit", "main") ++ path.path)
+    .addParam("message", s"Update $helpId")
+}
+
+object HelpBody {
+  type Props = HelpBody
+
+  @Lenses
+  final case class State(content: Pot[String])
+
+  def load(uri: Uri)(implicit cs: ContextShift[IO]): IO[Try[String]] = {
+    val backend = FetchCatsBackend[IO]()
+    basicRequest
+      .get(uri)
+      .readTimeout(5.seconds)
+      .send(backend)
+      .map {
+        _.body.leftMap(s => new RuntimeException(s)).toTry
+      }
+  }
+
+  private val component =
+    ScalaComponent
+      .builder[Props]
+      .initialState(State(Pot.pending))
+      .render_PS { (p, s) =>
+        val imageConv = (s: Uri) => p.baseUrl.addPath(s.path)
+        AppCtx.using { implicit ctx =>
+          HelpCtx.usingView { helpCtx =>
+            val helpView = helpCtx.zoom(HelpContext.displayedHelp)
+            val editUrl  = s.content match {
+              case Ready(_) => p.editPage
+              case _        => p.newPage
+            }
+            <.div(
+              ExploreStyles.HelpSidebar,
+              <.div(
+                ExploreStyles.HelpTitle,
+                <.h4(ExploreStyles.HelpTitleLabel, "Help"),
+                <.div(
+                  Button(as = <.a,
+                         size = Mini,
+                         compact = true,
+                         onClick = helpView.set(None).runAsyncCB
+                  )(
+                    Icons.Edit.size(Small).fitted(true)
+                  )(^.href := editUrl.toString(), ^.target := "_blank"),
+                  Button(size = Mini, compact = true, onClick = helpView.set(None).runAsyncCB)(
+                    Icons.Close.size(Small).fitted(true)
+                  )
+                )
+              ),
+              <.div(
+                ExploreStyles.HelpBody,
+                <.div(
+                  ExploreStyles.HelpBody,
+                  s.content match {
+                    case Ready(a)         =>
+                      ReactMarkdown(content = a,
+                                    clazz = ExploreStyles.HelpMarkdownBody,
+                                    linkTarget = "_blank",
+                                    imageConv
+                      ): VdomNode
+                    case Pending(_)       => <.div(ExploreStyles.HelpMarkdownBody, "Loading...")
+                    case crystal.Error(_) =>
```

Comment:
Maybe we want to discriminate on the error.

It seems that if there's a timeout for example, then the user will get "Not found" and the option to create the page.

Reply:
I'm not sure what to offer in case of timeout, just an error message, or maybe a retry button?

Reply:
Maybe we can just discriminate the "not found" error and in all other cases show a generic error?

The retry button sounds good too, although if the user closes and opens the help panel it will work as a retry right?